From 5a11ed4b7b897364d606c00b272c727105d77ad7 Mon Sep 17 00:00:00 2001
From: AlexAlt <alexandra.m.altieri@gmail.com>
Date: Thu, 19 Nov 2015 10:14:42 -0800
Subject: [PATCH 04/16] Add require csv to application.rb

---
 code_challenge/config/application.rb | 1 +
 1 file changed, 1 insertion(+)

diff --git a/code_challenge/config/application.rb b/code_challenge/config/application.rb
index 8499225..45f04a4 100644
--- a/code_challenge/config/application.rb
+++ b/code_challenge/config/application.rb
@@ -9,6 +9,7 @@ require "action_controller/railtie"
 require "action_mailer/railtie"
 require "action_view/railtie"
 require "sprockets/railtie"
+require 'csv'
 # require "rails/test_unit/railtie"
 
 # Require the gems listed in Gemfile, including any gems
-- 
2.3.2 (Apple Git-55)

